app/models/user.rb

```diff
@@ -55,8 +40,16 @@ def send_reminder_message
 
     @client.account.messages.create(from: ENV['FROM_TWILIO_NUMBER'],
                                     to: number,
-                                    body: last)
+                                    body: next_message)
+    increment_conversation
```

Comment:
If we do this...
then every 24hrs the reminder will have a different message.

Lets make a test of this.